ikarus/python/test/testdirichletvalues.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+import debug_info
+
+debug_info.setDebugFlags()
+
+import ikarus as iks
+import dune.grid
+import dune.functions
+
+import math
+
+
+def makeGrid():
+    lowerLeft = []
+    upperRight = []
+    elements = []
+    for _ in range(2):
+        lowerLeft.append(-1)
+        upperRight.append(1)
+        elements.append(3)
+
+    return dune.grid.structuredGrid(lowerLeft, upperRight, elements)
+
+
+def testDirichletValues():
+    grid = makeGrid()
+    basis = iks.basis(grid, dune.functions.Power(dune.functions.Lagrange(order=1), 2))
+    basis = basis.flat()
+
+    dirichletValues = iks.dirichletValues(basis)
+
+    assert basis.size() == dirichletValues.size
+
+    def fixOneIndex(vec, globalIndex):
+        if globalIndex == 0:
```

Comment:
Here this relies on the fact that globalIndex is implicitly convertible to int, from MultIndex of size 1, isn't it?